types.d.ts

```diff
@@ -0,0 +1,55 @@
+export type MontiEvent = any
+export type KadiraInfo = any
+
+export type ConnectOptions = {
+    enableErrorTracking?: boolean
+    endpoint?: string
+    hostname?: string
+    uploadSourceMaps?: boolean
+    recordIPAddress?: string
+    eventStackTrace?: boolean
+    disableNtp?: boolean
+    stalledTimeout?: number
+}
+
+export type TrackErrorOptions = {
+    type?: string
+    subType?: string
+    kadiraInfo?: KadiraInfo
+}
+
+export type TraceInfo = {
+    type: string
```

Comment:
There are only 3 values for the trace type: https://github.com/monti-apm/monti-apm-agent/blob/912d2623bc5961c2b6ce3665494edafe40072ba6/lib/tracer/tracer.js#L7